microsoft/testsuites/xdp/xdptools.py

```diff
@@ -41,7 +41,7 @@ class XdpTool(Tool):
     """
 
     _xdp_tools_repo = "https://github.com/xdp-project/xdp-tools.git"
-    _xdp_tools_tag = "v1.2.0"
+    _xdp_tools_tag = "v1.4.1"
```

Comment:
any reason to change the version

Reply:
Found a subtest failure due to a bug in older xdp version. It passed with the newer version.